python/nano/src/bigdl/nano/pytorch/trainer/Trainer.py

```diff
@@ -104,3 +108,24 @@ def __init__(self, num_processes: int = 1,
 
             super().__init__(accelerator=accelerator,
                              plugins=[plugin], *args, **kwargs)
+
+    @staticmethod
+    def compile(model: nn.Module, loss: _Loss = None, optimizer: torch.optim = None):
+        """
+        Compile a pytorch model into a pytorch-lightning model and return it.
```

Comment:
the comment is incorrect

Reply:
Please review modified docstring. Any further suggestions to properly describe this function?